src/UI/templates/default/Icon/icon.less

```diff
@@ -0,0 +1,67 @@
+.icon {
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-image: url('./images/icon_default.svg');
```

Comment:
Just realized, that we only need to use one variable all the way through "background-images-path". See comment bellow to @nhaagen.